vms/proposervm/metrics/metrics.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package metrics
```

Comment:
Why do we define our own package for this? These are service-level metrics, should these live in the same package as the service?

Reply:
I didn't write this file, but I imagine it was written out as a stub for any metrics we wanted to include in the proposervm. It's copied over from `https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/metrics/metrics.go`

@michaelkaplan13 not sure if we had any particular metrics in mind we wanted to track. Otherwise I can remove this file.